code/datums/abnormality/_ego_datum/teth.dm

```diff
@@ -179,3 +179,8 @@
 /datum/ego_datum/weapon/bean
 	item_path = /obj/item/ego_weapon/mini/bean
 	cost = 20
+
+// My Sweet Home - Dwelling
```

Comment:
Don't forget, hearth must have its own datum in addition to dwelling. The armor you choose will also need datum as well.

Reply:
Kirie provided an example of Feather of Honour(or something similar) and Paradise. Those two are one weapon, therefore only have one entry in the datum. This is the same as I had cannibalised the code for the two weapons.